TASVideos/WikiModules/PublicationsByPlatform.cshtml

```diff
@@ -0,0 +1,13 @@
+﻿@model PublicationsByPlatform
+<table>
+	@foreach (var platform in Model.Platforms.OrderBy(res => res.DisplayName))
+	{
+		<tr>
+			<td>@platform.DisplayName</td>
+			<td><a href="/Movies-@platform.Code">All Publications</a></td>
+			<td><a href="/Movies-@platform.Code-Standard">Standard</a></td>
+			<td><a href="/Movies-@platform.Code-Stars">Stars</a></td>
+			<td><a href="/Movies-@platform.Code-Alternative">Alternative</a></td>
```

Comment:
Since classes can be edited using site UI (and we recently added a new class), I suggest getting the list of existing classes and iterating though it, instead of hardcoding their then-names.